src/coreclr/jit/lower.cpp

```diff
@@ -5697,7 +5784,7 @@ bool Lowering::LowerUnsignedDivOrMod(GenTreeOp* divMod)
         }
         else
         {
-#ifdef TARGET_ARM64
+#if defined(TARGET_ARM64) //|| defined(TARGET_LOONGARCH64)
```

Comment:
Is this commented out on purpose? If so, remove the comment

```suggestion
#if defined(TARGET_ARM64)
```